2024-11-20 00:12:50.099 +06:00 [INF] Test log entry outside middleware
2024-11-20 00:12:50.543 +06:00 [INF] Now listening on: http://localhost:5109
2024-11-20 00:12:50.544 +06:00 [INF] Application started. Press Ctrl+C to shut down.
2024-11-20 00:12:50.545 +06:00 [INF] Hosting environment: Development
2024-11-20 00:12:50.545 +06:00 [INF] Content root path: /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.WebAPI
2024-11-20 00:12:50.777 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/index.html - null null
2024-11-20 00:12:50.942 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/index.html - 200 null text/html;charset=utf-8 166.7749ms
2024-11-20 00:12:50.999 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/swagger-ui-standalone-preset.js - null null
2024-11-20 00:12:50.999 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/swagger-ui.css - null null
2024-11-20 00:12:50.999 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/swagger-ui-bundle.js - null null
2024-11-20 00:12:51.131 +06:00 [INF] Sending file. Request path: '/swagger-ui-standalone-preset.js'. Physical path: 'N/A'
2024-11-20 00:12:51.131 +06:00 [INF] Sending file. Request path: '/swagger-ui.css'. Physical path: 'N/A'
2024-11-20 00:12:51.131 +06:00 [INF] Sending file. Request path: '/swagger-ui-bundle.js'. Physical path: 'N/A'
2024-11-20 00:12:51.133 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/swagger-ui.css - 200 143943 text/css 133.739ms
2024-11-20 00:12:51.133 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/swagger-ui-bundle.js - 200 1096145 text/javascript 133.7509ms
2024-11-20 00:12:51.133 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/swagger-ui-standalone-preset.js - 200 339486 text/javascript 133.7454ms
2024-11-20 00:12:51.510 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - null null
2024-11-20 00:12:51.593 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - 200 null application/json;charset=utf-8 82.4464ms
2024-11-20 00:13:42.311 +06:00 [INF] Request starting HTTP/1.1 POST http://localhost:5109/User - application/json 168
2024-11-20 00:13:42.325 +06:00 [INF] Logging Middleware Entered
2024-11-20 00:13:42.326 +06:00 [INF] Request: POST /User
2024-11-20 00:13:42.328 +06:00 [WRN] Failed to determine the https port for redirect.
2024-11-20 00:13:42.329 +06:00 [INF] Executing endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:13:42.381 +06:00 [INF] Route matched with {action = "CreateUser", controller = "User"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] CreateUser(TaskFlow.Data.Entities.UserEntity) on controller TaskFlow.WebAPI.Controllers.UserController (TaskFlow.WebAPI).
2024-11-20 00:13:42.397 +06:00 [INF] Executed action TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI) in 7.0544ms
2024-11-20 00:13:42.399 +06:00 [INF] Executed endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:13:42.509 +06:00 [ERR] An error occurred: Unable to resolve service for type 'TaskFlow.Services.UserService' while attempting to activate 'TaskFlow.WebAPI.Controllers.UserController'.
2024-11-20 00:13:42.554 +06:00 [ERR] An unhandled exception has occurred while executing the request.
System.ObjectDisposedException: Cannot access a closed Stream.
   at System.IO.MemoryStream.Write(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.MemoryStream.WriteAsync(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at System.IO.Pipelines.StreamPipeWriter.FlushAsyncInternal(Boolean writeToStream, ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask`1.GetTaskForValueTaskSource(IValueTaskSource`1 t)
--- End of stack trace from previous location ---
   at TaskFlow.Middlewares.ExceptionHandlingMiddleware.InvokeAsync(HttpContext context) in /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.Middlewares/ExceptionHandlingMiddleware.cs:line 28
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:13:42.616 +06:00 [ERR] An exception was thrown attempting to display the error page.
System.ObjectDisposedException: Cannot access a closed Stream.
   at System.IO.MemoryStream.Write(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.MemoryStream.WriteAsync(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at System.IO.Pipelines.StreamPipeWriter.FlushAsyncInternal(Boolean writeToStream, ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask`1.GetTaskForValueTaskSource(IValueTaskSource`1 t)
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.DisplayExceptionContent(ErrorContext errorContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:13:42.619 +06:00 [ERR] Connection id "0HN88ON3Q0ESE", Request id "0HN88ON3Q0ESE:00000003": An unhandled exception was thrown by the application.
System.ObjectDisposedException: Cannot access a closed Stream.
   at System.IO.MemoryStream.Write(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.MemoryStream.WriteAsync(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at System.IO.Pipelines.StreamPipeWriter.FlushAsyncInternal(Boolean writeToStream, ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask`1.GetTaskForValueTaskSource(IValueTaskSource`1 t)
--- End of stack trace from previous location ---
   at TaskFlow.Middlewares.ExceptionHandlingMiddleware.InvokeAsync(HttpContext context) in /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.Middlewares/ExceptionHandlingMiddleware.cs:line 28
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)
2024-11-20 00:13:42.622 +06:00 [INF] Request finished HTTP/1.1 POST http://localhost:5109/User - 500 0 null 310.9126ms
2024-11-20 00:14:29.955 +06:00 [INF] Request starting HTTP/1.1 POST http://localhost:5109/User - application/json 168
2024-11-20 00:14:29.956 +06:00 [INF] Logging Middleware Entered
2024-11-20 00:14:29.956 +06:00 [INF] Request: POST /User
2024-11-20 00:14:29.956 +06:00 [INF] Executing endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:14:29.957 +06:00 [INF] Route matched with {action = "CreateUser", controller = "User"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] CreateUser(TaskFlow.Data.Entities.UserEntity) on controller TaskFlow.WebAPI.Controllers.UserController (TaskFlow.WebAPI).
2024-11-20 00:14:29.958 +06:00 [INF] Executed action TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI) in 0.6136ms
2024-11-20 00:14:29.959 +06:00 [INF] Executed endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:14:30.026 +06:00 [ERR] An error occurred: Unable to resolve service for type 'TaskFlow.Services.UserService' while attempting to activate 'TaskFlow.WebAPI.Controllers.UserController'.
2024-11-20 00:14:30.060 +06:00 [ERR] An unhandled exception has occurred while executing the request.
System.ObjectDisposedException: Cannot access a closed Stream.
   at System.IO.MemoryStream.Write(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.MemoryStream.WriteAsync(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at System.IO.Pipelines.StreamPipeWriter.FlushAsyncInternal(Boolean writeToStream, ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask`1.GetTaskForValueTaskSource(IValueTaskSource`1 t)
--- End of stack trace from previous location ---
   at TaskFlow.Middlewares.ExceptionHandlingMiddleware.InvokeAsync(HttpContext context) in /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.Middlewares/ExceptionHandlingMiddleware.cs:line 28
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:14:30.062 +06:00 [ERR] An exception was thrown attempting to display the error page.
System.ObjectDisposedException: Cannot access a closed Stream.
   at System.IO.MemoryStream.Write(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.MemoryStream.WriteAsync(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at System.IO.Pipelines.StreamPipeWriter.FlushAsyncInternal(Boolean writeToStream, ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask`1.GetTaskForValueTaskSource(IValueTaskSource`1 t)
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.DisplayExceptionContent(ErrorContext errorContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:14:30.063 +06:00 [ERR] Connection id "0HN88ON3Q0ESE", Request id "0HN88ON3Q0ESE:00000004": An unhandled exception was thrown by the application.
System.ObjectDisposedException: Cannot access a closed Stream.
   at System.IO.MemoryStream.Write(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.MemoryStream.WriteAsync(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at System.IO.Pipelines.StreamPipeWriter.FlushAsyncInternal(Boolean writeToStream, ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask`1.GetTaskForValueTaskSource(IValueTaskSource`1 t)
--- End of stack trace from previous location ---
   at TaskFlow.Middlewares.ExceptionHandlingMiddleware.InvokeAsync(HttpContext context) in /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.Middlewares/ExceptionHandlingMiddleware.cs:line 28
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)
2024-11-20 00:14:30.064 +06:00 [INF] Request finished HTTP/1.1 POST http://localhost:5109/User - 500 0 null 109.7272ms
2024-11-20 00:14:48.309 +06:00 [INF] Request starting HTTP/1.1 POST http://localhost:5109/User - application/json 168
2024-11-20 00:14:48.310 +06:00 [INF] Logging Middleware Entered
2024-11-20 00:14:48.310 +06:00 [INF] Request: POST /User
2024-11-20 00:14:48.310 +06:00 [INF] Executing endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:14:48.310 +06:00 [INF] Route matched with {action = "CreateUser", controller = "User"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] CreateUser(TaskFlow.Data.Entities.UserEntity) on controller TaskFlow.WebAPI.Controllers.UserController (TaskFlow.WebAPI).
2024-11-20 00:14:48.311 +06:00 [INF] Executed action TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI) in 0.6295ms
2024-11-20 00:14:48.312 +06:00 [INF] Executed endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:14:48.381 +06:00 [ERR] An error occurred: Unable to resolve service for type 'TaskFlow.Services.UserService' while attempting to activate 'TaskFlow.WebAPI.Controllers.UserController'.
2024-11-20 00:14:48.414 +06:00 [ERR] An unhandled exception has occurred while executing the request.
System.ObjectDisposedException: Cannot access a closed Stream.
   at System.IO.MemoryStream.Write(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.MemoryStream.WriteAsync(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at System.IO.Pipelines.StreamPipeWriter.FlushAsyncInternal(Boolean writeToStream, ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask`1.GetTaskForValueTaskSource(IValueTaskSource`1 t)
--- End of stack trace from previous location ---
   at TaskFlow.Middlewares.ExceptionHandlingMiddleware.InvokeAsync(HttpContext context) in /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.Middlewares/ExceptionHandlingMiddleware.cs:line 28
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:14:48.417 +06:00 [ERR] An exception was thrown attempting to display the error page.
System.ObjectDisposedException: Cannot access a closed Stream.
   at System.IO.MemoryStream.Write(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.MemoryStream.WriteAsync(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at System.IO.Pipelines.StreamPipeWriter.FlushAsyncInternal(Boolean writeToStream, ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask`1.GetTaskForValueTaskSource(IValueTaskSource`1 t)
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.DisplayExceptionContent(ErrorContext errorContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:14:48.417 +06:00 [ERR] Connection id "0HN88ON3Q0ESE", Request id "0HN88ON3Q0ESE:00000005": An unhandled exception was thrown by the application.
System.ObjectDisposedException: Cannot access a closed Stream.
   at System.IO.MemoryStream.Write(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.MemoryStream.WriteAsync(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at System.IO.Pipelines.StreamPipeWriter.FlushAsyncInternal(Boolean writeToStream, ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask`1.GetTaskForValueTaskSource(IValueTaskSource`1 t)
--- End of stack trace from previous location ---
   at TaskFlow.Middlewares.ExceptionHandlingMiddleware.InvokeAsync(HttpContext context) in /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.Middlewares/ExceptionHandlingMiddleware.cs:line 28
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)
2024-11-20 00:14:48.419 +06:00 [INF] Request finished HTTP/1.1 POST http://localhost:5109/User - 500 0 null 110.0522ms
2024-11-20 00:14:49.265 +06:00 [INF] Request starting HTTP/1.1 POST http://localhost:5109/User - application/json 168
2024-11-20 00:14:49.266 +06:00 [INF] Logging Middleware Entered
2024-11-20 00:14:49.266 +06:00 [INF] Request: POST /User
2024-11-20 00:14:49.266 +06:00 [INF] Executing endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:14:49.266 +06:00 [INF] Route matched with {action = "CreateUser", controller = "User"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] CreateUser(TaskFlow.Data.Entities.UserEntity) on controller TaskFlow.WebAPI.Controllers.UserController (TaskFlow.WebAPI).
2024-11-20 00:14:49.267 +06:00 [INF] Executed action TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI) in 0.4461ms
2024-11-20 00:14:49.268 +06:00 [INF] Executed endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:14:49.336 +06:00 [ERR] An error occurred: Unable to resolve service for type 'TaskFlow.Services.UserService' while attempting to activate 'TaskFlow.WebAPI.Controllers.UserController'.
2024-11-20 00:14:49.370 +06:00 [ERR] An unhandled exception has occurred while executing the request.
System.ObjectDisposedException: Cannot access a closed Stream.
   at System.IO.MemoryStream.Write(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.MemoryStream.WriteAsync(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at System.IO.Pipelines.StreamPipeWriter.FlushAsyncInternal(Boolean writeToStream, ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask`1.GetTaskForValueTaskSource(IValueTaskSource`1 t)
--- End of stack trace from previous location ---
   at TaskFlow.Middlewares.ExceptionHandlingMiddleware.InvokeAsync(HttpContext context) in /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.Middlewares/ExceptionHandlingMiddleware.cs:line 28
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:14:49.372 +06:00 [ERR] An exception was thrown attempting to display the error page.
System.ObjectDisposedException: Cannot access a closed Stream.
   at System.IO.MemoryStream.Write(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.MemoryStream.WriteAsync(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at System.IO.Pipelines.StreamPipeWriter.FlushAsyncInternal(Boolean writeToStream, ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask`1.GetTaskForValueTaskSource(IValueTaskSource`1 t)
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.DisplayExceptionContent(ErrorContext errorContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:14:49.373 +06:00 [ERR] Connection id "0HN88ON3Q0ESE", Request id "0HN88ON3Q0ESE:00000006": An unhandled exception was thrown by the application.
System.ObjectDisposedException: Cannot access a closed Stream.
   at System.IO.MemoryStream.Write(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.MemoryStream.WriteAsync(ReadOnlyMemory`1 buffer, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at System.IO.Pipelines.StreamPipeWriter.FlushAsyncInternal(Boolean writeToStream, ReadOnlyMemory`1 data, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at System.Threading.Tasks.ValueTask`1.GetTaskForValueTaskSource(IValueTaskSource`1 t)
--- End of stack trace from previous location ---
   at TaskFlow.Middlewares.ExceptionHandlingMiddleware.InvokeAsync(HttpContext context) in /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.Middlewares/ExceptionHandlingMiddleware.cs:line 28
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)
2024-11-20 00:14:49.374 +06:00 [INF] Request finished HTTP/1.1 POST http://localhost:5109/User - 500 0 null 109.0396ms
2024-11-20 00:16:11.100 +06:00 [INF] Test log entry outside middleware
2024-11-20 00:16:11.511 +06:00 [INF] Now listening on: http://localhost:5109
2024-11-20 00:16:11.511 +06:00 [INF] Application started. Press Ctrl+C to shut down.
2024-11-20 00:16:11.512 +06:00 [INF] Hosting environment: Development
2024-11-20 00:16:11.512 +06:00 [INF] Content root path: /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.WebAPI
2024-11-20 00:16:11.672 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/index.html - null null
2024-11-20 00:16:11.784 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/index.html - 200 null text/html;charset=utf-8 113.2864ms
2024-11-20 00:16:12.228 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - null null
2024-11-20 00:16:12.323 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - 200 null application/json;charset=utf-8 94.8288ms
2024-11-20 00:16:21.531 +06:00 [INF] Request starting HTTP/1.1 POST http://localhost:5109/User - application/json 168
2024-11-20 00:16:21.549 +06:00 [WRN] Failed to determine the https port for redirect.
2024-11-20 00:16:21.550 +06:00 [INF] Executing endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:16:21.603 +06:00 [INF] Route matched with {action = "CreateUser", controller = "User"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] CreateUser(TaskFlow.Data.Entities.UserEntity) on controller TaskFlow.WebAPI.Controllers.UserController (TaskFlow.WebAPI).
2024-11-20 00:16:21.620 +06:00 [INF] Executed action TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI) in 6.0476ms
2024-11-20 00:16:21.621 +06:00 [INF] Executed endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:16:21.622 +06:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'TaskFlow.Services.UserService' while attempting to activate 'TaskFlow.WebAPI.Controllers.UserController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.ThrowHelperUnableToResolveService(Type type, Type requiredBy)
   at lambda_method11(Closure, IServiceProvider, Object[])
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass6_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|7_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:16:21.674 +06:00 [INF] Request finished HTTP/1.1 POST http://localhost:5109/User - 500 null text/plain; charset=utf-8 143.3194ms
2024-11-20 00:30:52.760 +06:00 [INF] Test log entry outside middleware
2024-11-20 00:30:53.135 +06:00 [INF] Now listening on: http://localhost:5109
2024-11-20 00:30:53.136 +06:00 [INF] Application started. Press Ctrl+C to shut down.
2024-11-20 00:30:53.136 +06:00 [INF] Hosting environment: Development
2024-11-20 00:30:53.137 +06:00 [INF] Content root path: /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.WebAPI
2024-11-20 00:30:53.295 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/index.html - null null
2024-11-20 00:30:53.404 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/index.html - 200 null text/html;charset=utf-8 110.1909ms
2024-11-20 00:30:53.747 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - null null
2024-11-20 00:30:53.831 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - 200 null application/json;charset=utf-8 84.1844ms
2024-11-20 00:31:00.022 +06:00 [INF] Request starting HTTP/1.1 POST http://localhost:5109/User - application/json 130
2024-11-20 00:31:00.045 +06:00 [WRN] Failed to determine the https port for redirect.
2024-11-20 00:31:00.047 +06:00 [INF] Executing endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:31:00.083 +06:00 [INF] Route matched with {action = "CreateUser", controller = "User"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] CreateUser(TaskFlow.Data.Entities.UserEntity) on controller TaskFlow.WebAPI.Controllers.UserController (TaskFlow.WebAPI).
2024-11-20 00:31:00.102 +06:00 [INF] Executed action TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI) in 5.8524ms
2024-11-20 00:31:00.103 +06:00 [INF] Executed endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:31:00.105 +06:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'TaskFlow.Services.UserService' while attempting to activate 'TaskFlow.WebAPI.Controllers.UserController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.ThrowHelperUnableToResolveService(Type type, Type requiredBy)
   at lambda_method11(Closure, IServiceProvider, Object[])
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass6_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|7_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:31:00.172 +06:00 [INF] Request finished HTTP/1.1 POST http://localhost:5109/User - 500 null text/plain; charset=utf-8 149.9782ms
2024-11-20 00:31:47.207 +06:00 [INF] Request starting HTTP/1.1 POST http://localhost:5109/User - application/json 130
2024-11-20 00:31:47.208 +06:00 [INF] Executing endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:31:47.209 +06:00 [INF] Route matched with {action = "CreateUser", controller = "User"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] CreateUser(TaskFlow.Data.Entities.UserEntity) on controller TaskFlow.WebAPI.Controllers.UserController (TaskFlow.WebAPI).
2024-11-20 00:31:47.210 +06:00 [INF] Executed action TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI) in 0.458ms
2024-11-20 00:31:47.210 +06:00 [INF] Executed endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:31:47.211 +06:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'TaskFlow.Services.UserService' while attempting to activate 'TaskFlow.WebAPI.Controllers.UserController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.ThrowHelperUnableToResolveService(Type type, Type requiredBy)
   at lambda_method11(Closure, IServiceProvider, Object[])
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass6_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|7_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:31:47.213 +06:00 [INF] Request finished HTTP/1.1 POST http://localhost:5109/User - 500 null text/plain; charset=utf-8 6.3305ms
2024-11-20 00:32:46.959 +06:00 [INF] Test log entry outside middleware
2024-11-20 00:32:47.347 +06:00 [INF] Now listening on: http://localhost:5109
2024-11-20 00:32:47.348 +06:00 [INF] Application started. Press Ctrl+C to shut down.
2024-11-20 00:32:47.348 +06:00 [INF] Hosting environment: Development
2024-11-20 00:32:47.348 +06:00 [INF] Content root path: /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.WebAPI
2024-11-20 00:32:47.507 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/index.html - null null
2024-11-20 00:32:47.622 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/index.html - 200 null text/html;charset=utf-8 115.2743ms
2024-11-20 00:32:47.991 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - null null
2024-11-20 00:32:48.075 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - 200 null application/json;charset=utf-8 83.6051ms
2024-11-20 00:33:11.901 +06:00 [INF] Request starting HTTP/1.1 POST http://localhost:5109/User - application/json 167
2024-11-20 00:33:11.919 +06:00 [WRN] Failed to determine the https port for redirect.
2024-11-20 00:33:11.921 +06:00 [INF] Executing endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:33:11.955 +06:00 [INF] Route matched with {action = "CreateUser", controller = "User"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] CreateUser(TaskFlow.Data.Entities.UserEntity) on controller TaskFlow.WebAPI.Controllers.UserController (TaskFlow.WebAPI).
2024-11-20 00:33:18.968 +06:00 [ERR] Failed executing DbCommand (103ms) [Parameters=[@p0='?' (DbType = Int32), @p1='?' (DbType = DateTime2), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 450)], CommandType='"Text"', CommandTimeout='30']
SET IMPLICIT_TRANSACTIONS OFF;
SET NOCOUNT ON;
INSERT INTO [Users] ([Id], [CreatedDate], [PasswordHash], [Role], [Username])
VALUES (@p0, @p1, @p2, @p3, @p4);
2024-11-20 00:33:19.004 +06:00 [ERR] An exception occurred in the database while saving changes for context type 'TaskFlow.Data.TaskFlowDbContext'.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert explicit value for identity column in table 'Users' when IDENTITY_INSERT is set to OFF.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:4a306522-d595-4582-a396-b50322a2b736
Error Number:544,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert explicit value for identity column in table 'Users' when IDENTITY_INSERT is set to OFF.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:4a306522-d595-4582-a396-b50322a2b736
Error Number:544,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
2024-11-20 00:33:19.104 +06:00 [INF] Executed action TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI) in 7139.9695ms
2024-11-20 00:33:19.104 +06:00 [INF] Executed endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:33:19.106 +06:00 [ERR] An unhandled exception has occurred while executing the request.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert explicit value for identity column in table 'Users' when IDENTITY_INSERT is set to OFF.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:4a306522-d595-4582-a396-b50322a2b736
Error Number:544,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at TaskFlow.Repositories.UserRepository.CreateUser(UserEntity user) in /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.Repositories/UserRepository.cs:line 33
   at TaskFlow.Services.UserService.CreateUser(UserEntity user) in /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.Services/UserService.cs:line 30
   at TaskFlow.WebAPI.Controllers.UserController.CreateUser(UserEntity user) in /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.WebAPI/Controllers/UserController.cs:line 50
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|7_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:33:19.140 +06:00 [INF] Request finished HTTP/1.1 POST http://localhost:5109/User - 500 null text/plain; charset=utf-8 7239.7123ms
2024-11-20 00:45:49.005 +06:00 [INF] Request starting HTTP/1.1 POST http://localhost:5109/User - application/json 156
2024-11-20 00:45:49.007 +06:00 [INF] Executing endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:45:49.008 +06:00 [INF] Route matched with {action = "CreateUser", controller = "User"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] CreateUser(TaskFlow.Data.Entities.UserEntity) on controller TaskFlow.WebAPI.Controllers.UserController (TaskFlow.WebAPI).
2024-11-20 00:45:51.877 +06:00 [INF] Executed DbCommand (24ms) [Parameters=[@p0='?' (DbType = DateTime2), @p1='?' (Size = 4000), @p2='?' (Size = 4000), @p3='?' (Size = 450)], CommandType='"Text"', CommandTimeout='30']
SET IMPLICIT_TRANSACTIONS OFF;
SET NOCOUNT ON;
INSERT INTO [Users] ([CreatedDate], [PasswordHash], [Role], [Username])
OUTPUT INSERTED.[Id]
VALUES (@p0, @p1, @p2, @p3);
2024-11-20 00:45:51.935 +06:00 [INF] Executing CreatedAtActionResult, writing value of type 'TaskFlow.Data.Entities.UserEntity'.
2024-11-20 00:45:51.976 +06:00 [INF] Executed action TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI) in 2967.9615ms
2024-11-20 00:45:51.976 +06:00 [INF] Executed endpoint 'TaskFlow.WebAPI.Controllers.UserController.CreateUser (TaskFlow.WebAPI)'
2024-11-20 00:45:51.977 +06:00 [INF] Request finished HTTP/1.1 POST http://localhost:5109/User - 201 null application/json; charset=utf-8 2971.8855ms
2024-11-20 00:46:32.053 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/User/john_doe89 - null null
2024-11-20 00:46:32.056 +06:00 [ERR] An unhandled exception has occurred while executing the request.
Microsoft.AspNetCore.Routing.Matching.AmbiguousMatchException: The request matched multiple endpoints. Matches: 

TaskFlow.WebAPI.Controllers.UserController.GetUserById (TaskFlow.WebAPI)
TaskFlow.WebAPI.Controllers.UserController.GetUserByUsername (TaskFlow.WebAPI)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ReportAmbiguity(Span`1 candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ProcessFinalCandidates(HttpContext httpContext, Span`1 candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.Select(HttpContext httpContext, Span`1 candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DfaMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:46:32.059 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/User/john_doe89 - 500 null text/plain; charset=utf-8 5.7288ms
2024-11-20 00:52:59.203 +06:00 [INF] Test log entry outside middleware
2024-11-20 00:52:59.598 +06:00 [INF] Now listening on: http://localhost:5109
2024-11-20 00:52:59.599 +06:00 [INF] Application started. Press Ctrl+C to shut down.
2024-11-20 00:52:59.599 +06:00 [INF] Hosting environment: Development
2024-11-20 00:52:59.599 +06:00 [INF] Content root path: /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.WebAPI
2024-11-20 00:52:59.769 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/index.html - null null
2024-11-20 00:52:59.879 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/index.html - 200 null text/html;charset=utf-8 110.6192ms
2024-11-20 00:53:00.242 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - null null
2024-11-20 00:53:00.329 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - 200 null application/json;charset=utf-8 86.9127ms
2024-11-20 00:53:21.103 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/User/john_doe89 - null null
2024-11-20 00:53:21.115 +06:00 [ERR] An unhandled exception has occurred while executing the request.
Microsoft.AspNetCore.Routing.Matching.AmbiguousMatchException: The request matched multiple endpoints. Matches: 

TaskFlow.WebAPI.Controllers.UserController.GetUserById (TaskFlow.WebAPI)
TaskFlow.WebAPI.Controllers.UserController.GetUserByUsername (TaskFlow.WebAPI)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ReportAmbiguity(Span`1 candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ProcessFinalCandidates(HttpContext httpContext, Span`1 candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.Select(HttpContext httpContext, Span`1 candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DfaMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:53:21.167 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/User/john_doe89 - 500 null text/plain; charset=utf-8 64.4623ms
2024-11-20 00:56:06.336 +06:00 [INF] Test log entry outside middleware
2024-11-20 00:56:06.711 +06:00 [INF] Now listening on: http://localhost:5109
2024-11-20 00:56:06.712 +06:00 [INF] Application started. Press Ctrl+C to shut down.
2024-11-20 00:56:06.713 +06:00 [INF] Hosting environment: Development
2024-11-20 00:56:06.713 +06:00 [INF] Content root path: /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.WebAPI
2024-11-20 00:56:06.881 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/index.html - null null
2024-11-20 00:56:07.005 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/index.html - 200 null text/html;charset=utf-8 125.0332ms
2024-11-20 00:56:07.415 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - null null
2024-11-20 00:56:07.509 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - 200 null application/json;charset=utf-8 94.3689ms
2024-11-20 00:56:14.107 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/User/1 - null null
2024-11-20 00:56:14.120 +06:00 [ERR] An unhandled exception has occurred while executing the request.
Microsoft.AspNetCore.Routing.Matching.AmbiguousMatchException: The request matched multiple endpoints. Matches: 

TaskFlow.WebAPI.Controllers.UserController.GetUserById (TaskFlow.WebAPI)
TaskFlow.WebAPI.Controllers.UserController.GetUserByUsername (TaskFlow.WebAPI)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ReportAmbiguity(Span`1 candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ProcessFinalCandidates(HttpContext httpContext, Span`1 candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.Select(HttpContext httpContext, Span`1 candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DfaMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2024-11-20 00:56:14.170 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/User/1 - 500 null text/plain; charset=utf-8 62.9115ms
2024-11-20 00:57:39.970 +06:00 [INF] Test log entry outside middleware
2024-11-20 00:57:40.360 +06:00 [INF] Now listening on: http://localhost:5109
2024-11-20 00:57:40.361 +06:00 [INF] Application started. Press Ctrl+C to shut down.
2024-11-20 00:57:40.362 +06:00 [INF] Hosting environment: Development
2024-11-20 00:57:40.362 +06:00 [INF] Content root path: /home/akib/Basic/Documents/WindowsTransfer/Dotnet/TaskFlow/TaskFlow-WebApi/TaskFlow.WebAPI
2024-11-20 00:57:40.524 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/index.html - null null
2024-11-20 00:57:40.632 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/index.html - 200 null text/html;charset=utf-8 109.7339ms
2024-11-20 00:57:40.987 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - null null
2024-11-20 00:57:41.073 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/swagger/v1/swagger.json - 200 null application/json;charset=utf-8 85.3274ms
2024-11-20 00:57:47.458 +06:00 [INF] Request starting HTTP/1.1 GET http://localhost:5109/api/User/id/1 - null null
2024-11-20 00:57:47.476 +06:00 [WRN] Failed to determine the https port for redirect.
2024-11-20 00:57:47.477 +06:00 [INF] Executing endpoint 'TaskFlow.WebAPI.Controllers.UserController.GetUserById (TaskFlow.WebAPI)'
2024-11-20 00:57:47.510 +06:00 [INF] Route matched with {action = "GetUserById", controller = "User"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] GetUserById(Int32) on controller TaskFlow.WebAPI.Controllers.UserController (TaskFlow.WebAPI).
2024-11-20 00:57:49.647 +06:00 [INF] Executed DbCommand (44ms) [Parameters=[@__p_0='?' (DbType = Int32)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [u].[Id], [u].[CreatedDate], [u].[PasswordHash], [u].[Role], [u].[Username]
FROM [Users] AS [u]
WHERE [u].[Id] = @__p_0
2024-11-20 00:57:49.703 +06:00 [INF] Executing OkObjectResult, writing value of type 'TaskFlow.Data.Entities.UserEntity'.
2024-11-20 00:57:49.715 +06:00 [INF] Executed action TaskFlow.WebAPI.Controllers.UserController.GetUserById (TaskFlow.WebAPI) in 2196.8488ms
2024-11-20 00:57:49.715 +06:00 [INF] Executed endpoint 'TaskFlow.WebAPI.Controllers.UserController.GetUserById (TaskFlow.WebAPI)'
2024-11-20 00:57:49.722 +06:00 [INF] Request finished HTTP/1.1 GET http://localhost:5109/api/User/id/1 - 200 null application/json; charset=utf-8 2263.588ms
